### ISSUE TITLE
Catch rate issue

### ISSUE THREAD

**Issue by NeptuneCykes** [Ajarmar/universal-pokemon-randomizer-zx#29]
While doing a run of Pokemon Y, I found out that when changing the minimum catch rate and making alternate forms of pokemon show up in the wild, those alternate forms do not get that change in their catch rate and instead get their vanilla catch rate, so making some pokemons really hard to get or basically impossible to catch due to being legendary pokemons or pokemons that normally have a really bad catch rate for it.

TLDR, changed catch rate doesn't apply to Alternate forms found in the wild, tested in Pokemon Y

**Comment by RevanXVIV** [comment]
This is a known issue. What pokemon did you have issue catching? I've caught Megas with no issue, Primals however, very much gave me issue.

**Comment by NeptuneCykes** [comment]
Someone had problems with Kyurem-White, so I went back and tried with it and couldn't catch it, then went to try a different altered form and went after Mewtwo-Y and happened the same. I'm guessing some Megas are gonna be easier to catch since their catch rate is not as high as Starter or Legendaries' catch rate, which are incredibly low so hard to catch.

**Comment by tom-overton** [comment]
Yep, this is issue #24 and has already been fixed with 6346435006bd2e865742a29d9c26e2344b2db366. It will be included in the next version of the randomizer. Sorry about the issue!
